database/users/db.go

```diff
@@ -24,6 +24,7 @@ const (
          						                	WHERE plugin_id = '86DC3AE7-9F3D-42CB-85A3-A71ABC3C3CB8'
          						                    AND attribute_name = 'wallet'
          						                    AND value->'wallet' ? $1);`
+	userExistsByNameQuery       = `SELECT EXISTS(SELECT * FROM "user" WHERE profile->>'name' = $1);`
```

Comment:
I'm not sure about Postgres optimizations, but maybe `... EXISTS(SELECT 1 ...` is better?